content/hardware/04.pro/carriers/portenta-hat-carrier/tutorials/user-manual/content.md

```diff
@@ -1805,15 +1806,25 @@ The following table shows an in-depth connector designation:
 |       13       |      N/A       |               |             N/A              |                                                                      |
 |       14       |      N/A       |               |           ETH_LED1           |                                J1-17                                 |
 
-Ethernet performance differs based on the associated Portenta board:
+Ethernet connection speeds differ based on the associated Portenta board:
 
 - With the Portenta X8: The system supports 1 Gbit Ethernet.
 - When combined with the Portenta H7 or C33: The performance is limited to 100 Mbit Ethernet.
 
-To configure the Ethernet settings, depending on the paired Portenta board, one must use the provided DIP switch located on the Portenta Hat Carrier. For an in-depth understanding of the DIP switch, kindly refer to [this section](#dip-switch-configuration).
+To configure the Ethernet settings, depending on the paired Portenta board, one must use the provided DIP switch on the Portenta Hat Carrier. The following table shows the specific DIP switch configuration needed to enable Ethernet on the carrier:
 
-#### Ethernet Interface With Linux
+| **Mounted Portenta Device** | **ETH CENTER TAP DIP SWITCH** |
+|:---------------------------:|:-----------------------------:|
+|         Portenta X8         |         Position: OFF         |
+|       Portenta H7/C33       |          Position: ON         |
+
+***For an in-depth understanding of the DIP switch, kindly refer to [this section](#dip-switch-configuration).***
 
+It is advisable to connect the Portenta X8 through the Portenta HAT Carrier to a device with DHCP server capabilities, such as a network router, to ease the automatic assignment of an IP address.
+
+It will allow the Portenta X8 to communicate with other devices on the network without manual IP configuration. Employing DHCP simplifies device management, supports dynamic reconfiguration, and provides an advantage for applications involving many devices.
```

Comment:
They are part of the same message so they should be part of the same paragrapth

```suggestion
It is advisable to connect the Portenta X8 through the Portenta HAT Carrier to a device with DHCP server capabilities, such as a network router, to ease the automatic assignment of an IP address. DHCP will allow the Portenta X8 to communicate with other devices on the network without manual IP configuration. Employing DHCP simplifies device management, supports dynamic reconfiguration, and provides an advantage for applications involving many devices.

In case you want to assign a manual IP to your device, or even create a direct network between your computer and your board, you can follow the multiple procedures available depending on your network devices and operating system.
```